### ISSUE TITLE
porter build fails & switch context message is wrong

### ISSUE THREAD

**Issue by agardnerIT** [getporter/porter#3028]
Following the "[getting started](https://porter.sh/docs/getting-started/create-bundle/#create-a-bundle)" guide to create a bundle and `porter build` fails.

The suggested fix is also wrong. It should be:

```
docker context use default
```

### Logs

```
% porter build
Copying porter runtime ===> 
Copying mixins ===> 
Copying mixin exec ===> 
Building invocation image
use `docker --context=default buildx` to switch to context "default"
unable to build CNAB invocation image: use `docker --context=default buildx` to switch to context "default"
unable to build CNAB invocation image: use `docker --context=default buildx` to switch to context "default"
```

### Porter Version

```
% porter version
porter v1.0.14 (0e739d88)
```

### Fix

Running this works:
```
docker context use default
porter build
```

**Comment by nwsteenberg** [comment]
This seems to be an issue with the Docker buildx builder lib:
https://github.com/docker/buildx/blob/8abef5908705e49f7ba88ef8c957e1127b597a2a/builder/builder.go#L132

**Comment by nwsteenberg** [comment]
Error is originating from buildx lib: [buildx/builder/builder.go](https://github.com/docker/buildx/blob/8abef5908705e49f7ba88ef8c957e1127b597a2a/builder/builder.go#L132)
 I agree that it might not make sense in a Porter context, and the suggested fix will not do anything.

**Comment by schristoff** [comment]
This should be fixed in versions 1.0.16+ - could you try in the newer versions and let me know if you're still seeing it?

**Comment by schristoff** [comment]
Hey! We haven't heard from you in awhile, but please reopen an issue if you're seeing this in the newer versions. :)
